gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/uikit/internal/registerConnectedDeviceTasks.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.experimental.uikit.internal
+
+import org.gradle.api.*
+import org.jetbrains.compose.desktop.application.internal.MacUtils
+import org.jetbrains.compose.experimental.dsl.DeployTarget
+import org.jetbrains.compose.experimental.uikit.tasks.AbstractComposeIosTask
+
+fun Project.registerConnectedDeviceTasks(
+    id: String,
+    deploy: DeployTarget.ConnectedDevice,
+    projectName: String,
+    bundleIdPrefix: String
+) {
+    val xcodeProjectDir = getBuildIosDir(id).resolve("$projectName.xcodeproj")
+    val iosCompiledAppDir = xcodeProjectDir.resolve("build/Build/Products/Debug-iphoneos/$projectName.app")
```

Comment:
Guess we shall select Debug/Release based on fun parameter.

Reply:
Yes. It is my mistake! I will fix it, Thanks!

Reply:
Done